spec/requests/posts_spec.rb

```diff
@@ -2,16 +2,38 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
+    before(:each) do
+      get '/users/3/posts'
+    end
+
     it 'returns http success' do
-      get '/posts/index'
       expect(response).to have_http_status(:success)
     end
+
+    it 'if correct template rendered' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'response body includes correct placeholder' do
+      expect(response.body).to include("Here's a List of posts for a given user!!")
+    end
   end
 
   describe 'GET /show' do
-    it 'returns http success' do
-      get '/posts/show'
+    before(:each) do
+      get '/users/3/posts/1'
+    end
+
+    it 'response status correct' do
```

Comment:
- [ ] Great effort working on project milestones (i.e. `controller-spec`), But currently, I notice some of the test cases are failing in both the `user` and `post` spec. I would kindly suggest you write a correct test case and be sure they are all passing.

Below is the video recording 👇 


https://github.com/Othniel2471/blog_app/assets/78634841/461b90d9-5a6e-4087-9f18-74a6fef0a3a5

Reply:
@brainconnect93  thank you for the review, but all the tests are passing on my end.